### DISCUSSION TITLE
Caching a react craco app with turbo repo.

### DISCUSSION THREAD

**Discussion by onlit** [vercel/turborepo#9539] [Help]
### Summary

I have a monorepo that has many next.js apps but one react app which uses craco. After doing a bit research, I couldn't find any resources nor examples on how to cache react apps with turbo repo. Is that not supported or just not well documented?

This is how I've set up my turbo repo config for now:

```
{
  "$schema": "https://turbo.build/schema.json",
  "ui": "tui",
  "tasks": {
    "build": {
      "dependsOn": ["^build"],
      "inputs": ["$TURBO_DEFAULT$", ".env*"],
      "outputs": [".next/**", "!.next/cache/**"]
    },
    "admin-react#build": {
      "dependsOn": ["^build"],
      "inputs": ["$TURBO_DEFAULT$", ".env*"],
      "outputs": ["build/**"]
    },
    "lint": {
      "dependsOn": ["^lint"],
      "inputs": ["$TURBO_DEFAULT$"]
    },
    "dev": {
      "cache": false,
      "persistent": true
    },
    "start": {
      "cache": false
    },
    "clean": {
      "cache": false
    }
  }
}
```

**Comment by anthonyshew** [comment]
Turborepo works with any framework (or language, in actuality). If I remember correctly, the output directory for `create-react-app` is indeed `build`, so your configuration should be correct. Are you experiencing issues?
